apis/system/tests/common.py

```diff
@@ -0,0 +1,39 @@
+import subprocess
+
+
+def to_R(x):
```

Comment:
sorry, _what_ is  being `to_R`'ed? i _think_ this is something that convertions python booleans (and maybe other things) to R string -- please let's either have the function name say something about what it does, and/or a doc-block

Reply:
Basically this is to inject a Python object in the auto-generated R script. If we want to be explicit, this could be `embed_into_R_code` or something like that.